app/views/codequest/Frame.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="frame">
+    <Heading />
```

Comment:
we shouldn't be using single name as Vue module name, even eslint throws an error. Have you not set it up?